webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -290,6 +303,34 @@ protected function setLastEvent(?string $eventId): void
             ->execute();
     }
 
+    protected function setLastHTTPCode(int $httpCode): void
+    {
+        // We use a direct query since we need to reload the source otherwise and we don't want that since that would
```

Comment:
I don't get why / where we need to reload the entity to set (or get) the code. And even so, this one extra single query to set the HTTP code on reconnect doesn't seem too much compared to all the queries the event feed reader does?

Reply:
It was copied from above, so I assumed the same reasoning (and didn't fully understand why we didn't want to reload above).